src/arc_utilities/ros_helpers.py

```diff
@@ -59,3 +60,25 @@ def __str__(self):
 def logfatal(exception_class, msg):
     rospy.logfatal(msg)
     raise exception_class(msg)
+
+
+def get_oneshot_publisher(topic_path: str, *args, **kwargs):
```

Comment:
Rename to `get_connected_publisher`. The returned publisher does not just have to be used for a oneshot application